launcher/Cargo.toml

```
@@ -9,7 +9,7 @@ homepage.workspace = true
 [dependencies]
 clap = { version = "4.4.5", features = ["derive", "env"] }
 ctrlc = { version = "3.4.1", features = ["termination"] }
-nix = "0.27.1"
+nix = "0.28.0"
```

thanks for the contribution. I believe the `signal` is needed with the new version

```suggestion
nix = { version = "0.28.0", features = ["signal"] }
```